dashboard/dashboard.sh

```diff
@@ -36,6 +37,9 @@ export JOB_API_TOKEN_PROD=""
 export AWS_ACCESS_KEY_ID=""
 export AWS_SECRET_ACCESS_KEY=""
 export S3_REGION_NAME="us-east-1"
+export AUTH_DOMAIN="go-ready-blockchain.firebaseapp.com"
+export DATABASEURL="https://go-ready-blockchain.firebaseio.com"
+export STORAGE_BUCKET="go-ready-blockchain.appspot.com"
 
 function run_smoketests()
```

Comment:
Dead code, can be removed

Reply:
i have commented the vars because we might need it if we generate the SLA part